Classes/vendor/MYUtilities/MYErrorUtils.m

```diff
@@ -139,22 +139,19 @@ BOOL MYMiscError( NSError **error, NSString *message, ... )
 #if !TARGET_OS_IPHONE || defined(__SEC_TYPES__)
     else if ($equal(domain,NSOSStatusErrorDomain)) {
         // If it's an OSStatus, check whether CarbonCore knows its name:
-        const char *name = NULL;
-#if !TARGET_OS_IPHONE
-        name = GetMacOSStatusErrorString((int)code);
-#endif
-        if (name && *name)
-            result = [NSString stringWithCString: name encoding: NSMacOSRomanStringEncoding];
-        else {
+        NSError *osErr = [NSError errorWithDomain:NSOSStatusErrorDomain code:code userInfo:nil];
+        result = [osErr localizedDescription];
```

Comment:
Actually.. playing around with this a bit, here is what I've learned, build the following CLI application:
```objc
#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        // insert code here...
        NSLog(@"Hello, World!");

        int code = fnfErr;

        const char *name = GetMacOSStatusErrorString(code);
        NSString *orig = [NSString stringWithCString: name encoding: NSMacOSRomanStringEncoding];
        NSLog(@"orig: %@", orig);

        NSError *osErr = [NSError errorWithDomain:NSOSStatusErrorDomain code:code userInfo:nil];
        NSString *loc = [osErr localizedDescription];
        NSLog(@"loc: %@", loc);

        NSString *desc = [osErr description];
        NSLog(@"desc: %@\n", desc);

    }
    return 0;
}
```
The result is:
```shell
2015-06-03 09:28:02.320 oserr[14220:3115096] Hello, World!
2015-06-03 09:28:02.321 oserr[14220:3115096] orig: fnfErr
2015-06-03 09:28:02.324 oserr[14220:3115096] loc: The operation couldn’t be completed. (OSStatus error -43.)
2015-06-03 09:28:02.325 oserr[14220:3115096] desc: Error Domain=NSOSStatusErrorDomain Code=-43 "The operation couldn’t be completed. (OSStatus error -43.)" (fnfErr: File not found)
```

So the result is significantly different.

If this makes anyone nervous, we can resort to squelching the warning with a `#pragma`.